keras_cv/datasets/waymo/load.py

```diff
@@ -16,24 +16,29 @@
 
 import tensorflow as tf
 import tensorflow_datasets as tfds
-from tensorflow_datasets.proto.waymo_dataset_generated_pb2 import Frame
+from waymo_open_dataset import dataset_pb2
+
+from keras_cv.datasets.waymo import transformer
 
 
 def _generate_frames(segments, transformer):
     def _generator():
         for record in tfds.as_numpy(segments):
-            frame = Frame()
+            frame = dataset_pb2.Frame()
             frame.ParseFromString(record)
             yield transformer(frame)
 
     return _generator
 
 
-def load(tfrecord_path, transformer, output_signature):
+def load(
+    tfrecord_path,
+    transformer=transformer.build_tensors_from_wod_frame,
```

Comment:
Let's also update the docstring to reflect this default transformer and output signature
Other than that LGTM.

Thank you!

Reply:
Done, thanks Ian!